opensuse-base-image/src/main/docker/Dockerfile

```diff
@@ -64,7 +64,7 @@ COPY --from=stage2 /usr/local/bin/su-exec /usr/local/bin/su-exec
 COPY startup /startup
 ADD https://raw.githubusercontent.com/CAFapi/caf-common/v1.19.0/container-cert-script/install-ca-cert.sh \
     /startup/startup.d/install-ca-cert-base.sh
-RUN chmod +x /startup/* /startup/startup.d/*
+RUN chmod -R a+rwx /startup
```

Comment:
@andyreidz Is write access required here?  Should it be just read and execute?

Reply:
When running as root the container would have been able to write here, so when running as another user or group I thought this would be the most compatible. Happy to try a+rx and see if there are issues.

Reply:
Let's try `a+rx` then.  I can't think why anyone would we writing to the startup folder so if that does cause issues it would be good to see what they are.